app.js

```diff
@@ -38,9 +38,103 @@ function operate(operator, a, b) {
 const display = document.querySelector(".display");
 const btns = document.querySelectorAll("button");
 
-// put number selected on the display
 btns.forEach((button) => {
-  button.addEventListener("click", (e) => {
-    display.innerHTML += button.innerText;
-  });
+  button.addEventListener("click", (e) => calculatorDisplay(button.id));
 });
+
+let firstNum = "";
+let secondNum = "";
+let operator;
+// put number selected on the display
+function calculatorDisplay(value) {
+  function getData(value) {
```

Comment:
This function is poorly named: usually anything starting with "get" is a "getter" and should return a value. In this case it would make more sense to name it something like "addDigit"